src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
I don't use new or this; as an extension, neither does HyperApp or will ever do. So, why should Router imply that?